ovos_number_parser/numbers_sl.py

```
@@ -0,0 +1,462 @@
+    1e18: 'kvintiljonti',
+    1e21: 'sekstilijonti',
+    1e24: 'septilijonti',
+    1e27: 'oktilijonti',
+    1e30: 'nonilijonti',
+    1e33: 'decilijonti'
+    # TODO > 1e33
+}
+_SHORT_ORDINAL_SL.update(_ORDINAL_BASE_SL)
+
+
+def nice_number_sl(number, speech=True, denominators=range(1, 21)):
```

Ruff

Do not perform function call `range` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)